sdk/instrumentation/net/http/handler.go

```diff
@@ -87,16 +88,25 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 		defer r.Body.Close()
 
+		isMultipartFormDataBody := HasMultiPartFormDataContentTypeHeader(headerMapAccessor{r.Header})
+
 		// Only records the body if it is not empty and the content type
 		// header is not streamable
 		if len(body) > 0 {
-			setTruncatedBodyAttribute("request", body, int(h.dataCaptureConfig.BodyMaxSizeBytes.Value), span)
+			setTruncatedBodyAttribute("request", body, int(h.dataCaptureConfig.BodyMaxSizeBytes.Value), span,
```

Comment:
How does this work, the setTrucatedBodyAttribute has 4 arguments.

Reply:
Nevermind, that is confusing.  The logic looks correct.  There are 2 of those functions with the same name
`bodyattribute.setTruncatedBodyAttribute` and `body.setTruncatedBodyAttribute`.

minor: Perhaps it could all be combined into `body.setBodyAttributes()`.

Reply:
they should all be in different packages.